enhancements/metering/slim-install.md

```diff
@@ -0,0 +1,281 @@
+
+### Continue Reconciling the Hive and Presto Components
+
+This alternative is centered around building off of what is being proposed in this enhancement and providing an avenue for Metering to reconcile Hive and Presto components to scale out to larger (or busier) cluster environments. With this approach, all transactional type operations can be done in Postgres (i.e. the creation of tables and the insertion of new metric values) but all analytical querying will be done using Presto. This implementation would require deploying Presto with the [Postgres connector enabled](https://prestosql.io/docs/current/connector/postgresql.html) and updating the Hive metastore configuration to always use Postgres as the underlying database.
+
+While this approach would provide an a potential avenue for scaling out this new, proposed Metering architecture, it would also increase the supportability matrix and the complexity of the stack, in addition to having to continue maintaining those additional components.
+
+### Integrate the Presto/Prometheus Connector
+
+This distinct alternative is focused around extending the Hadoop-based Metering stack and integrating the Presto/Prometheus connector. This connector, which was [introduced upstream in the 334 release](https://github.com/prestosql/presto/pull/2321), allows Presto to directly query Prometheus metric data. Now, instead of periodically querying Prometheus for metric query results, the connector will stream metric samples from Prometheus, to database tables stored in Presto.
+
+This approach would allow Metering to interact with more raw metric data, in addition to a more streamlined data flow. The process of integrating this connector will require a full overhaul of the existing promQL and SQL queries/views defined throughout the default set of `ReportDataSource` and `ReportQuery` resources, in addition to rethinking how storage would need to be handled.
+
+In theory, Metering can utilize existing CRD functionality to periodically store daily partitions of Prometheus data in Presto schemas. One avenue that could be explored is the usage of scheduled `Reports`, which Metering stores streamed Prometheus data in daily partitions in Presto. Alternatively, a `INSERT INTO ... SELECT * FROM ... WHERE timestamp > .status.lastTimestamp` type operation could be entertained and achieve similar results.
+
+Once that initial integration has happened, it's possible to remove (i.e. stop reconciling) the Hive components (metastore and server). Presto, an in-memory query engine, still requires the usage of a metastore, but it's possible to have Presto use a local, file-based metastore and back up that path in the container with a PersistentVolumeClaim (PVC). With this path, the reporting-operator would always create tables in Presto instead of Hive, i.e. always favor creating `PrestoTable` resources, and the functionality provided by the Hive components would be obsolete.
+
```

Comment:
What specific storage can be used as an alternative to metastore for storing the db metadata?

Reply:
I'm not sure I follow the context of what you meant by "specific storage" here. We currently spin up a PVC for the /var/lib/hive metastore database, and we would likely do something similar, but instead mount a path in the Presto container responsible for acting and storing as a metastore.